includes/google_scholar.inc

```diff
@@ -44,36 +43,46 @@ function islandora_scholar_create_meta_tags($object) {
     else {
       return FALSE;
     }
-    foreach ($mods_xml->xpath('mods:name') as $name_xml) {
+    foreach ($mods_xml->xpath(variable_get('islandora_scholar_xpaths_authors_xpath', 'mods:name')) as $name_xml) {
       $name_parts = array();
       $name_xml->registerXPathNamespace('mods', 'http://www.loc.gov/mods/v3');
-      $roles = $name_xml->xpath('mods:role/mods:roleTerm');
```

Comment:
This is incorrect behavior for reasons too long to list as part of the code review, instead I'll post it in the PR thread.

Reply:
The problem here is everyone call its "author" roleTerm differently. We use "Creator(s)". That's why original unmodified module didn't work for us. I was assuming in the admin panel, you will set the XPath to the actual spot for the author. Otherwise, we may need to add a spot in admin panel to let user customized what roleTerm they are looking for.